install/zowe-install.sh

```diff
@@ -148,6 +148,23 @@ cp $INSTALL_DIR/scripts/ocopyshr.sh $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.sh
 cp $INSTALL_DIR/scripts/ocopyshr.clist $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.clist
 echo "Copying the run-zowe.sh into "$ZOWE_ROOT_DIR/scripts/internal >> $LOG_FILE
 
+# Create the /fingerprint directory in the ZOWE_ROOT_DIR runtime directory,
+# if it exists in the INSTALL_DIR driectory
+if [[ -d $INSTALL_DIR/fingerprint ]]
+then
+  echo "OK: Fingerprint exists"
```

Comment:
Remove debug echo

Reply:
I find it useful to have this message written to the console output of the Jenkins run.  Granted, it will appear to the user too, which is perhaps negative.  It's logged too, but the log is not printed by Jenkins unless the build goes wrong IIRC.  Maybe I should echo only if fingerprint does not exist, which is what happens in install-packaging to create the fingerprint.

Reply:
The log is printed in the build always.
I didn't object to the "no fingerprint" echo as that should only shows in the build, but this ok message will show to users and that's not appropriate for debug messages

Reply:
Removed.